README.md

```diff
@@ -24,7 +24,7 @@ Clone the [repository](https://github.com/mozilla-services/pageshot/).
 There are two scripts to run the server locally and develop the add-on:
 
 - `./bin/run-server` will run the server on `http://localhost:10080` and automatically restart if there are changes.
-- `./bin/run-addon` will build a few parts of the addon (into `webextension/build/`) and start Firefox with the add-on installed.  The add-on will be refreshed automatically as you change files.  We recommend you open `about:debugging` to help debug the extension.
+- `./bin/run-addon` will build a few parts of the addon (into `addon/webextension/build/`) and start Firefox with the add-on installed.  The add-on will be refreshed automatically as you change files.  We recommend you open `about:debugging` to help debug the extension.
```

Comment:
I improved the docs here – `run-addon` still just runs webextension, but `run-addon --bootstrap` runs with the whole bootstrap wrapper.  You are right about about:debugging for that latter case.